rasa/utils/tensorflow/model_data.py

```diff
@@ -158,6 +158,13 @@ def __setstate__(self, state: Any, **kwargs: Any) -> None:
         self.units = state[-1]
         super(FeatureArray, self).__setstate__(state[0:-3], **kwargs)
 
+    def __str__(self) -> Text:
```

Comment:
If we only use this for debugging, I think `__repr__` would be better.

Reply:
with __repr__, I would expect the values to be printed as well which the current __repr__ does and it was nice to have that short representation... I'll add an additional __repr__ with everything and leave this __str__ as well